common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -32,11 +33,15 @@ class @Problem
     @checkButtonCheckText = @checkButtonLabel.text()
     @checkButtonCheckingText = @checkButton.data('checking')
     @checkButton.click @check_fd
+    @hintButton = @$('div.action button.hint-button')
+    @hintButton.click @hint_button
```

Comment:
i'm not familiar with this syntax — does this define what to do when `@hintButton` is clicked?